src/main/java/org/breedinginsight/brapps/importer/model/base/Study.java

```diff
@@ -41,6 +41,17 @@
     @ImportFieldMetadata(id="studyName", name="Study Name", description = "The name of the study.")
     private String studyName;
 
+    @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
+    @ImportMappingRequired
+    @ImportFieldMetadata(id="studyType", name="Study Type",
+            description = "Type of study, can be one of the predefined breedbase types or custom: " +
+                    "misc_trial, Preliminary Yield Trial, Uniform Yield Trial, heterosis_trial, " +
+                    "storage_trial, genetic_gain_trial, health_status_trial Seedling Nursery, " +
+                    "Variety Release Trial, Clonal Evaluation, Advanced Yield Trial, crossing_trial, " +
+                    "grafting_trial, pollinating_trial, phenotyping_trial, genotyping_trial, " +
+                    "Seed Multiplication, Screen House, crossing_block_trial, Specialty Trial, etc.")
+    private String studyType;
```

Comment:
I would remove `misc_trial` from the list. That's an internal breedbase thing.

Reply:
ya, I wasn't sure about that since it's showing up in the trial design as an option in the breedbase GUI. I grabbed all the terms from the db and looks like there's a few extra compared to the GUI.
![image](https://user-images.githubusercontent.com/53413353/129368426-60724370-3e85-440d-92c0-7a4d722add7c.png)

Reply:
Let me know if you still think I should remove it.

Reply:
Oh I see. Yeah I had to put in a trial type of misc_trial whose cvterm value is recorded as the type the user passes in. In brapi it returns whatever the user enters and doesn't show misc_trial. So I guess we could show it, I just wonder if it is useful to anyone. I guess these help texts will all need a pass through by Shawn to clean them up anyway, so you can probably just leave it for now.